manager/controllers/baseline_delete.go

```diff
@@ -63,5 +64,8 @@ func BaselineDeleteHandler(c *gin.Context) {
 		LogAndRespNotFound(c, errors.New("no rows returned"), "baseline not found")
 		return
 	}
+
+	kafka.EvaluateBaselineSystems(nil, account)
```

Comment:
Rather than recalculating all systems without baseline for an account I'd remember list of system before baseline is deleted and then recalculate just them.